lib/quicly.c

```diff
@@ -3327,10 +3317,8 @@ static int handle_ack_frame(quicly_conn_t *conn, size_t epoch, quicly_ack_frame_
     /* OnPacketAckedCC */
     /* TODO (jri): this function should be called for every packet newly acked. (kazuho) I do not think so;
      * quicly_loss_on_packet_acked is NOT OnPacketAcked */
-    if (smallest_newly_acked != UINT64_MAX)
-        quicly_loss_on_packet_acked(&conn->egress.loss, smallest_newly_acked);
-
     if (bytes_acked > 0) {
+        conn->egress.loss.pto_count = 0;
```

Comment:
This change removes the abstraction that was provided by `quicly_loss_on_packet_acked`. I do not have a strong preference on eliminating the abstraction (the reason I created the abstraction is because I am unfamiliar with loss recovery and therefore wanted a clean-cut API as much as possible), but wonder what your opinion is on how much split loss recovery and the transport logic should be.

Reply:
That is a good point. I wanted to clean up the abstraction a bit, but I'll add the abstraction back in now and do the cleanup later.